node/node.go

```diff
@@ -149,6 +151,10 @@ func makeHTTP(port uint16, rpcHandler *rpc.Handler, log utils.SimpleLogger) *jso
 	}, log)
 }
 
+func makeGRPC(port uint16, db db.DB, log utils.Logger) *grpc.Server {
+	return grpc.NewServer(port, db, log)
+}
+
 // Run starts Juno node by opening the DB, initialising services.
```

Comment:
we can inline this?

Reply:
sure, I just wanted to mimic http service with its `makeHTTP(...)`. inlined